package.json

```diff
@@ -109,6 +109,9 @@
     "@unstoppabledomains/resolution": "5.0.2",
     "@walletconnect/client": "1.6.6",
     "@walletconnect/utils": "1.6.6",
+    "appcenter": "4.4.3",
+    "appcenter-analytics": "4.4.3",
+    "appcenter-crashes": "4.4.3",
```

Comment:
are these two deps mandatory? Doesn't seem like we want them

Reply:
we do need the appcenter for sure, the other two are not necessary

Reply:
I got asked for installing them so I followed the steps. I can remove them if you insist

Reply:
removing